fs/fs_zenfs_test.cc

```diff
@@ -0,0 +1,49 @@
+#include "fs_zenfs.h"
+
+#include <gtest/gtest.h>
+
+#include <cstdlib>
+
+#include "rocksdb/convenience.h"
+#include "rocksdb/env.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+class FsZenFSTest : public testing::Test {
+ public:
+  std::shared_ptr<FileSystem> fs_;
+
+  FsZenFSTest() {}
+
+ protected:
+  void SetUp() override {
+    ConfigOptions config_options;
+    config_options.ignore_unsupported_options = false;
+    char* fs_uri_env = std::getenv("DEVICE");
+    ASSERT_TRUE(fs_uri_env != nullptr) << "DEVICE environment variable not set";
+    std::string fs_uri = "zenfs://dev:" + std::string(fs_uri_env);
+    Status s = FileSystem::CreateFromString(config_options, fs_uri, &fs_);
+    ASSERT_TRUE(s.ok()) << "Failed to create ZenFS instance";
+  }
+
+  void TearDown() override {}
+};
+
+TEST_F(FsZenFSTest, Example) {
+  IOStatus status;
+
+  ASSERT_TRUE(1);
+
+  IOOptions options;
+  status = fs_->CreateDir("test", options, nullptr);
```

Comment:
This creates the directory in aux-path. 
As an example we can add something happening within ZenFS. Like create a file maybe.

Reply:
> This creates the directory in aux-path.

Yes, that was the intention.

> As an example we can add something happening within ZenFS. Like create a file maybe.

For sure. This is just a placeholder to show that the infrastructure is working and how to add new test cases. I imagine that we want a whole bunch of tests to cover as much of ZenFS as possible.

Perhaps we should make a habit of adding a test each time we find a bug so that the test triggers said bug. Sort of TDD for bug fixing.